statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -1798,7 +1899,7 @@ def _cov_params_oim(self, approx_complex_step=True, approx_centered=False):
 
         (neg_cov, singular_values) = pinv_extended(evaluated_hessian)
 
-        self.model.update(self.params)
+        self.model.update(self.params, transformed=True, includes_fixed=True)
```

Comment:
why are transformed and includes_fixed hardcoded here (and elsewhere)?

Reply:
In the results object, it's because `self.params` is already in the space of the model (so it is already "transformed") and it has been already expanded to include the fixed parameters.